[vtk] Remove stack trace printing in Common::Core

Generically printing a stack trace induces a whole lot of complicated
support code that Drake neither wants nor needs.

--- Common/Core/vtkDeserializer.cxx
+++ Common/Core/vtkDeserializer.cxx
@@ -186,8 +186,10 @@ vtkDeserializer::ConstructorType vtkDeserializer::GetConstructor(
     name = superClassNames[--superClassId];
   } while (superClassId > 0);
   vtkErrorMacro(<< "There is no constructor registered for type " << className
-                << ". Check stack trace to see how we got here.");
+                << ".");
+#if 0
   vtkWarningMacro(<< vtksys::SystemInformation::GetProgramStack(2, 1));
+#endif
   return nullptr;
 }
 
@@ -218,8 +220,10 @@ vtkDeserializer::HandlerType vtkDeserializer::GetHandler(const std::type_info& t
        "{ .name="
     << type.name() << " .hashCode=" << type.hash_code()
     << " }"
-       " because a deserializer was not found. Check stack trace to see how we got here.");
+       " because a deserializer was not found.");
+#if 0
   vtkWarningMacro(<< vtksys::SystemInformation::GetProgramStack(2, 1));
+#endif
   return nullptr;
 }
 
diff --git a/Common/Core/vtkSerializer.cxx b/Common/Core/vtkSerializer.cxx
index 3d4233932a..400ab5de08 100644
--- Common/Core/vtkSerializer.cxx
+++ Common/Core/vtkSerializer.cxx
@@ -112,8 +112,10 @@ vtkSerializer::HandlerType vtkSerializer::GetHandler(const std::type_info& type)
        "{ .name="
     << type.name() << " .hashCode=" << type.hash_code()
     << " }"
-       " because a serializer was not found. Check stack trace to see how we got here.");
+       " because a serializer was not found.");
+#if 0
   vtkWarningMacro(<< vtksys::SystemInformation::GetProgramStack(2, 1));
+#endif
   return nullptr;
 }
 
